support/specsplit.js

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env node -r esm
```

Comment:
I moved this to the `package.json` as my bash couldn't handle the shebang properly